src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/BackgroundTaskContext.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using OrchardCore.Environment.Shell.Scope;
+
+namespace OrchardCore.BackgroundTasks
+{
+    public class BackgroundTaskContext
```

Comment:
```suggestion
    public class BackgroundTaskEventContext
```

Maybe?

Reply:
Idem here, because i thought we could also use it as a param of `DoWorkAsync()` if we plan to change its signature as said in my previous comment.

But as said, would be a breaking change to modify the `DoWorkAsync()` signature

So you're right, as for now it is only used for `IBackgroundTaskEventHandler`, better to name it `BackgroundTaskEventContext`